agents/random.py

```diff
@@ -20,13 +20,30 @@ def __init__(
     def forward(self, inputs: Dict) -> Dict:
         neighbors = inputs[self._neighbors_prefix]  # (batch_size, neighbors_cnt)
 
-        neighbors_mask = inputs[f'{self._neighbors_prefix}.mask']  # (batch_size, neighbors_cnt)
-        neighbors_mask_sum = torch.sum(neighbors_mask, dim=-1)  # (batch_size)
+        neighbors_length = neighbors.lengths  # (batch_size)
 
-        next_neighbors = torch.from_numpy(np.random.randint(
-            low=np.zeros_like(neighbors_mask_sum.cpu().detach().numpy()),
-            high=neighbors_mask_sum.cpu().detach().numpy()
-        )).to(neighbors.device)  # (batch_size)
+        low = torch.zeros_like(neighbors_length)  # (batch_size)
+        high = neighbors_length  # (batch_size)
+        next_neighbors = torch.round(torch.rand(neighbors_length.shape) * (neighbors_length - torch.ones(1))).long()  # (batch_size)
```

Comment:
```suggestion
        next_neighbors = torch.round(torch.rand(neighbors_length.shape) * (neighbors_length -1).long()  # (batch_size)
```

You don't need to write `torch.ones(1)`, you can use simple variables